src/Glpi/Form/ServiceCatalog/Provider/CategoryProvider.php

```diff
@@ -87,4 +88,53 @@ public function getItems(ItemRequest $item_request): array
 
         return $categories;
     }
+
+    /**
+     * @param ItemRequest $item_request
+     * @param bool $return_current Include the current category in the result
+     * @return array<array{id: int, name: string}>
+     */
+    public function getAncestors(ItemRequest $item_request, bool $return_current = true): array
```

Comment:
This parameter is never passed by the caller, no need to have a complex signature if it is not mandatory.

```suggestion
    public function getAncestors(ItemRequest $item_request): array
```